cmp/grid/columns/Column.js

```diff
@@ -190,14 +190,19 @@ export class Column {
                 headerComponentParams: {gridModel, column: this}
             };
 
+        // Our implementation of Grid.getDataPath() > Record.xhTreePath returns data path []s of
+        // Record IDs. TreeColumns use those IDs as their cell values, regardless of field.
+        // Add valueGetters below to correct + additional fixes for sorting below.
         if (this.isTreeColumn) {
             ret.showRowGroup = true;
             ret.cellRenderer = 'agGroupCellRenderer';
             ret.cellRendererParams = {
                 suppressCount: true,
                 suppressDoubleClickExpand: true,
-                innerRenderer: (v) => v.data[this.field]
+                innerRenderer: (v) => v.data[field]
```

Comment:
could definite a valueGetter variable to a single instance of thsi function on line 197, and just set that three times.  Obviously not critical  :-)